src/main.ts

```diff
@@ -196,6 +197,14 @@ function compile(config: webpack.Config, options: WebpackOptions): Promise<void>
 				}
 
 				console.log(stats.toString(options.stats));
+
+				if (stats.compilation && stats.compilation.errors && stats.compilation.errors.length > 0) {
+					reject({
+						exitCode: 1,
+						message: ''
```

Comment:
surely we can provide something more informative?

Reply:
Well.. I tried something more informative but it just looked silly. At this point the webpack build has already printed out all the errors so the console is full of the _real_ error details.  I had could try something like `The build failed with errors` or something, but like I said, it seemed obvious from the red text above it... @kitsonk thoughts?

Reply:
Hmm... slightly raises the question of if we have a `--force` or not... which then this message could indicate that there are compilation errors reported, use `--force` to ignore.

Reply:
I like that. I will add support for `--force`! Can I print out `use the --force to continue anyways`? 😛

Reply:
These aren't the 🤖 you are looking for.